moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -1,17 +1,18 @@
 <?xml version="1.0"?>
 <launch>
   <arg name="paused" default="false"/>
-  <arg name="gui" default="true"/>
+  <arg name="gazebo_gui" default="true"/>
+  <arg name="urdf_path" default="[URDF_LOCATION]"/>
 
   <!-- startup simulated world -->
   <include file="$(find gazebo_ros)/launch/empty_world.launch">
     <arg name="world_name" default="worlds/empty.world"/>
     <arg name="paused" value="$(arg paused)"/>
-    <arg name="gui" value="$(arg gui)"/>
+    <arg name="gui" value="$(arg gazebo_gui)"/>
   </include>
 
   <!-- send robot urdf to param server -->
-  <param name="robot_description" [URDF_LOAD_ATTRIBUTE] />
+  <param name="robot_description" textfile="$(arg urdf_path)" />
 
   <!-- push robot_description to factory and spawn robot in gazebo at the origin, change x,y,z arguments to spawn in a different position -->
   <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model robot -x 0 -y 0 -z 0"
```

Comment:
This file seems to introduce a run dependency on `gazebo_ros`, but the PR does not seem to update the `package.xml` template to include that dependency.

If that was on purpose (fi to keep the nr of dependencies down and make Gazebo support optional), then I believe it would be good to document that somewhere to make it explicit.